docs/api/cypress-api/require.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Cypress.require
+e2eSpecific: true
+---
+
+`Cypress.require` enables utilizing dependencies within the
+[`cy.origin()`](/api/commands/origin) callback function. It is used to require
+modules such as [npm](https://www.npmjs.com/) packages and other local files.
+
+It is functionally the same as using
+[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+in browser-targeted code.
+
+## Syntax
+
+```js
+Cypress.require(moduleId)
```

Comment:
```suggestion
Cypress.require(moduleId)
Cypress.require(relative_path)
```
I assume the path is relative to the spec file right?

Reply:
Not necessarily as it could be a [core module identifier, node_modules package identifier, an absolute path, etc](https://nodejs.org/api/modules.html#all-together). I changed it to `moduleNameOrPath`, since [that's what's Node's description of it is](https://nodejs.org/api/modules.html#requireid), but I'm open to other ideas.